### PR TITLE
Fixed json so that it is acceptable to the REST API

### PR DESCRIPTION
**Pull request by rajagopalans**

### CODE REVIEW

File: Interface_Flap/release_4.2.1/Content/probes/interface-flap.json [hunk 1]
```diff
@@ -84,131 +84,38 @@
...
-            },
             "units": {
                 "value": ""
             },
             "tags": [],
-            "anomaly_count": 0,
             "enable_metric_logging": false,
             "retention_duration": 2592000,
             "retention_size": 0,
             "graph_annotation_properties": {},
```
Comment (mehdi-juniper) [inline Interface_Flap/release_4.2.1/Content/probes/interface-flap.json:106]: last item in the object, remove `,`

File: Interface_Flap/release_4.2.1/Content/probes/interface-flap.json [hunk 2]
```diff
@@ -84,131 +84,38 @@
...
-            },
             "units": {
                 "value": ""
             },
             "tags": [],
-            "anomaly_count": 0,
             "enable_metric_logging": false,
             "retention_duration": 86400,
             "retention_size": 0,
             "graph_annotation_properties": {},
```
Comment (mehdi-juniper) [inline Interface_Flap/release_4.2.1/Content/probes/interface-flap.json:118]: last item in the object, remove `,`

File: Interface_Flap/release_4.2.1/Content/probes/interface-flap.json [hunk 3]
```diff
@@ -84,131 +84,38 @@
...
-            },
             "units": {
                 "value": ""
             },
             "tags": [],
-            "anomaly_count": 0,
             "enable_metric_logging": true,
             "retention_duration": 2592000,
             "retention_size": 0,
             "graph_annotation_properties": {},
```
Comment (mehdi-juniper) [inline Interface_Flap/release_4.2.1/Content/probes/interface-flap.json:94]: last item in the object, remove `,`

**Comment by mehdi-juniper** [comment]
just few trailing `,`, will merge after you update and then generalise these changes to all other examples. @rajagopalans

**Comment by rajagopalans** [comment]
Done


Juniper Business Use Only

________________________________

**Review by mehdi-juniper — APPROVED** [review]

**Pull request merged**